docs/_getting_started/jupyter.md

```diff
@@ -87,11 +216,14 @@ You can also launch interactive consoles from JupyterHub.  These will behave ver
 
 You can also launch a bash terminal from JupyterHub.  This provides access to commands you would normally run on the Yens, but through the web browser.
 
+!!! warning
+    This is not the same as the log in terminal via ssh.  You should not do the following: ssh, installing software, and evaluating the contents of /tmp
```

Comment:
I spoke to adam and tested a few things -- Let's reword to this:
> You may notice slight differences between the Jupyter terminal and a standard ssh login terminal. Please use a traditional ssh terminal to build, install and configure software. If you experience any unexpected behavior in the Jupyter terminal, please log in using ssh and try again.

Reply:
done